### PR TITLE
samples: matter: define sample persistence subscription configuration

### PR DESCRIPTION
**Pull request by doublemis1**
Definition of new configuration of light bulb matter sample and light switch matter sample needed to execute load memory tests.

### CODE REVIEW

File: samples/matter/light_bulb/sample.yaml [hunk 1]
```diff
@@ -47,3 +47,9 @@ tests:
     integration_platforms:
       - nrf5340dk_nrf5340_cpuapp
     platform_allow: nrf5340dk_nrf5340_cpuapp
+  # Sample to execute load tests
+  sample.matter.light_bulb.persistence_subscription:
```
Comment (jnsgsr) [inline samples/matter/light_bulb/sample.yaml:51]: You have to select integration platforms, if you don't want it to run in integration builds you have to use quarantine configuration file. Maybe enable one of them for 7002 and another for 52840. I think the tag is redundant for now but it is a good idea to comment configurations if they are specific in testing context.

**Review by jnsgsr — CHANGES_REQUESTED** [review]

**Review by kkasperczyk-no — APPROVED** [review]

File: samples/matter/light_switch/sample.yaml [hunk 2]
```diff
@@ -9,3 +9,11 @@ tests:
...
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
+  # Sample to execute load tests
+  sample.matter.light_switch.persistent_subscriptions:
+    build_only: true
+    extra_args: CONFIG_CHIP_PERSISTENT_SUBSCRIPTIONS=y
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
+    integration_platforms:
+      - nrf7002dk_nrf5340_cpuapp
+    tags: load
```
Comment (jnsgsr) [inline samples/matter/light_switch/sample.yaml:19]: the tag will not be used, it is redundant

File: samples/matter/light_bulb/sample.yaml [hunk 3]
```diff
@@ -12,3 +12,17 @@ tests:
...
+      - nrf5340dk_nrf5340_cpuapp
+    platform_allow: nrf5340dk_nrf5340_cpuapp
+  # Sample to execute load tests
+  sample.matter.light_bulb.persistent_subscriptions:
+    build_only: true
+    extra_args: CONFIG_CHIP_PERSISTENT_SUBSCRIPTIONS=y
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
+    integration_platforms:
+      - nrf52840dk_nrf52840
+    tags: load
```
Comment (jnsgsr) [inline samples/matter/light_bulb/sample.yaml:28]: Redundant

File: samples/matter/light_bulb/sample.yaml [hunk 4]
```diff
@@ -12,3 +12,17 @@ tests:
       - nrf5340dk_nrf5340_cpuapp
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
+  sample.matter.light_bulb.nrf7002_ek:
```
Comment (jnsgsr) [inline samples/matter/light_bulb/sample.yaml:15]: Should not be here I guess

**Review by jnsgsr — APPROVED** [review]

**Pull request merged**